### PR TITLE
Update changelog with bug fix

### PR DESCRIPTION
**Pull request by katthartic**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ It's a Blueprint-palooza this week, with lots of new data vendors to integrate w
 - The [HTTP Download File from URL](blueprint-library/http/http-download-file-from-url.md) Blueprint has been updated to accept custom headers so you can customize your requests.
 - The [Domo Blueprints](blueprint-library/domo/domo-overview.md) have been updated to accept developer tokens as an alternative to username/password.
 
+### Bug Fixes
+- Empty non-required password fields in Blueprint vessels were being rejected on Fleet save. They have now be authorized to remain empty if you so choose.
```
Comment (ericelsken) [inline docs/changelog.md:48]: What does "They have now be authorized to remain empty" mean?

**Review by ericelsken — CHANGES_REQUESTED** [review]

File: docs/changelog.md (hunk 1 above)
Comment (katthartic) [reply docs/changelog.md:48]: Just that you can leave non-required password fields empty.

Comment (ericelsken) [reply docs/changelog.md:48]: Okay. Should it be "have now been authorized"? Is "have now be" correct? [](edited-with-Axolo)

Comment (katthartic) [reply docs/changelog.md]: LOL missed that both times. It is now updated. [](edited-with-Axolo)

**Review by ericelsken — APPROVED** [review]

**Comment by JonD85** [comment]
Do I need to look at this somewhere?

**Comment by katthartic** [comment]
Nope. I got approval from Eric.

**Pull request merged**